selfdrive/ui/translations/main_nl.ts

```diff
@@ -0,0 +1,1305 @@
+    </message>
+    <message>
+        <location filename="../qt/maps/map_settings.cc" line="169"/>
+        <source>No work
+location set</source>
+        <translation>Geen werklocatie ingesteld</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map_settings.cc" line="282"/>
+        <source>no recent destinations</source>
+        <translation>Geen recente bestemmingen</translation>
+    </message>
+</context>
+<context>
+    <name>MapWindow</name>
+    <message>
+        <location filename="../qt/maps/map.cc" line="178"/>
+        <source>Map Loading</source>
+        <translation>Kaart wordt geladen</translation>
+    </message>
+    <message>
+        <location filename="../qt/maps/map.cc" line="195"/>
+        <source>Waiting for GPS</source>
+        <translation>Wachten op GPS</translation>
+    </message>
+</context>
+<context>
+    <name>MultiOptionDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="300"/>
+        <source>Select</source>
+        <translation>Selecteer</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="333"/>
+        <source>Cancel</source>
+        <translation>Annuleren</translation>
+    </message>
+</context>
+<context>
+    <name>Networking</name>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="30"/>
+        <source>Advanced</source>
+        <translation>Geavanceerd</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="87"/>
+        <source>Enter password</source>
+        <translation>Voer wachtwoord in</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="87"/>
+        <location filename="../qt/offroad/networking.cc" line="97"/>
+        <source>for &quot;%1&quot;</source>
+        <translation>voor &quot;%1&quot;</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/networking.cc" line="97"/>
+        <source>Wrong password</source>
+        <translation>Verkeerd wachtwoord</translation>
+    </message>
+</context>
+<context>
+    <name>NvgWindow</name>
+    <message>
+        <location filename="../qt/onroad.cc" line="212"/>
+        <source>km/h</source>
+        <translation>km/u</translation>
+    </message>
+    <message>
+        <location filename="../qt/onroad.cc" line="212"/>
+        <source>mph</source>
+        <translation>mph</translation>
+    </message>
+    <message>
+        <location filename="../qt/onroad.cc" line="280"/>
+        <location filename="../qt/onroad.cc" line="283"/>
+        <source>MAX</source>
+        <translation>MAX</translation>
+    </message>
+    <message>
+        <location filename="../qt/onroad.cc" line="327"/>
+        <location filename="../qt/onroad.cc" line="330"/>
+        <source>SPEED</source>
+        <translation>SNELHEID</translation>
```

Comment:
```suggestion
        <translation>SPEED</translation>
```
These represent the words "speed limit" in the US speed limit sign. They probably shouldn't be translated.